src/org/thoughtcrime/securesms/ConversationListActivity.java

```diff
@@ -282,7 +282,7 @@ public void openConversation(int chatId, int startingPosition) {
     final ApplicationDcContext dcContext = DcHelper.getContext(this);
     if (isForwarding(this) && dcContext.getChat(chatId).isSelfTalk()) {
       SendRelayedMessageUtil.immediatelyRelay(this, chatId);
-      Toast.makeText(this, "✔️ " + getString(R.string.saved), Toast.LENGTH_SHORT).show();
+      Toast.makeText(this, DynamicTheme.getCheckmarkEmoji(this) + "️ " + getString(R.string.saved), Toast.LENGTH_SHORT).show();
```

Comment:
this is not related to html-view, however, during review it turns out that the "✔️" emoji does not contrast nicely to dark-mode background-colors and we introduced a function therefore - and used it at places where  "✔️" was already used before.